plenum/server/primary_selector.py

```diff
@@ -54,14 +54,56 @@ def next_primary_node_name(self, instance_id, nodeReg=None):
 
         return name
 
-    def next_primary_replica_name(self, instance_id, nodeReg=None):
+    def next_primary_node_name_for_master(self, nodeReg=None):
```

Comment:
Looks like this method must be private (with _ prefix).

Reply:
Done.